akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -834,25 +837,38 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
 
   // InboundContext
   override def sendControl(to: Address, message: ControlMessage) =
-    association(to).sendControl(message)
-
-  override def send(message: Any, sender: OptionVal[ActorRef], recipient: RemoteActorRef): Unit = {
-    val cached = recipient.cachedAssociation
+    try {
+      association(to).sendControl(message)
+    } catch {
+      case ShuttingDown ⇒ // silence it
+    }
 
-    val a =
-      if (cached ne null) cached
-      else {
-        val a2 = association(recipient.path.address)
-        recipient.cachedAssociation = a2
-        a2
-      }
+  override def send(message: Any, sender: OptionVal[ActorRef], recipient: RemoteActorRef): Unit =
+    try {
+      val cached = recipient.cachedAssociation
+
+      val a =
+        if (cached ne null) cached
+        else {
+          val a2 = association(recipient.path.address)
+          recipient.cachedAssociation = a2
+          a2
+        }
 
-    a.send(message, sender, OptionVal.Some(recipient))
-  }
+      a.send(message, sender, OptionVal.Some(recipient))
+    } catch {
+      case ShuttingDown ⇒ // silence it
```

Comment:
If I'd move it in there, I'd have to make association return an `OptionVal` here plus check `isShutdown` on every message in `send` which I understood that we should avoid.